lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -21,6 +21,8 @@ no indirect;
 __PACKAGE__->register_type('consumer_groups');
 
 use constant RESPONSE_TIMEOUT => $ENV{RPC_QUEUE_RESPONSE_TIMEOUT} // 30;
+use constant DEFAULT_CATEGORY_NAME => 'general';
```

Comment:
Seems that this should be documented somewhere that it will default to `general`?

Reply:
We are just switching `mt5` calls in this stage and here: https://github.com/regentmarkets/binary-websocket-api/blob/master/lib/Binary/WebSocketAPI/Actions.pm#L89, we only set `msg_group` for mt5 methods. So to keep the change non-breaking and not requiring to set `general` for all other methods in `Actions.pm`, I thought it was pretty much obvious that the default value should be `general`. If this should get documented somewhere, please guide me on where to add this.

Reply:
Is this going to be a public module?

Reply:
This _is_ a public module: https://metacpan.org/pod/Mojo::WebSocketProxy

Its probably best to have for `ConsumerGroups->new` to accept a `msg_group` or `category` parameter that can fall back to `general`, so it is better documented here (and less associated with a specific category implementation.)

Reply:
I think it is currently a public module :thinking:

Reply:
@zakame I don't think we want to run category-specific websocket proxies! This fallback here is for the actions that don't have their `msg_group` set. So I don't see how passing a `msg_group` to `ConsumerGroups` constructor can help improve the situation.

Reply:
In any case the public will have no idea how we are implementing it so it needs needs to be documented that it will use `general` by default.

Reply:
@nima-fs my point is exactly what @michaelmueller-binary said: if its a publicly visible change, it needs to be documented.

(Also, need to refrain from discussing further specifics or linking to private repos here, again this is a public module!)

Reply:
I have deleted the comment that included link to private repos. Will add the documentation as well. @zakame @michaelmueller-binary